cli/flox/doc/flox-auth.md

```diff
@@ -0,0 +1,33 @@
+---
+title: FLOX-AUTH
+section: 1
+header: "flox User Manuals"
```

Comment:
```suggestion
header: "Flox User Manuals"
```

I dont know if this uses the brand name or the command name -- `man git` and `man curl` do not agree either.

Reply:
Fixed